testing/integration/fleetserver_fips_test.go

```diff
@@ -73,14 +73,16 @@ func TestFIPSAgentConnectingToFIPSFleetServerInECHFRH(t *testing.T) {
 	var agentIsFIPS bool
 	for _, item := range agents.Items {
 		if item.PolicyID == cloudAgentPolicyID {
+			t.Logf("Found fleet-server entry: %+v", item)
 			agentStatus = item.Status
 			agentIsFIPS = item.LocalMetadata.Elastic.Agent.FIPS
+			break
 		}
 	}
 
 	// Check that this Agent is online (i.e. healthy) and is FIPS-capable. This
 	// will prove that a FIPS-capable Agent is able to connect to a FIPS-capable
 	// Fleet Server, with both running in ECH.
-	require.Equal(t, "online", agentStatus)
 	require.Equal(t, true, agentIsFIPS)
+	require.Equal(t, "online", agentStatus)
```

Comment:
@ycombinator this test is currently failing; the instance deployed in ECH does not state it's a FIPs instance; can we confirm that the kibana API returns this information?

Also @kruskall, an earlier test where the status check was first reported a degraded status, i think this is due to the apm-server but have not yet recreated to test.

Reply:
>the instance deployed in ECH does not state it's a FIPs instance;

@michel-laterman  Is this expected?